### ISSUE TITLE
Home Assistant Core crashes several times a day

### ISSUE THREAD

**Issue by BernieV1977** [kcofoni/ha-netro-watering#23]
Hi,
The last days my HA Core installation (latest version 2024.8.2) crashes several times a day.
In the home-assistant.log.1 file I see that's the last lines always are the polling instructions of the Whisperer 2 sensors (I have in total 9 and polling is every 10 minutes).

After I disabled the polling of the sensors in HA then crashing of the system seems to be gone.
This week I upgrades the HACS integration to the latest version, could this be related? It started after upgrading HACS.
At this moment I don't have any logs to show here but could this be checked that the cause could be HACS?
Thx

Bernd

**Comment by BernieV1977** [comment]
Thx for your feedback. After a day or 2 without crashing HA after disable the polling of the Netro sensors I enabled it again without issues. Meanwhile there where some other integrations I updated and at this moment no problems anymore. I could be that this issue isn't Netro inegration related. Still on HA 2024.8.2, I update in the first days and let you know if this can be closed. Regards, 
Bernd
